matchzoo/engine/param_table.py

```diff
@@ -64,3 +73,20 @@ def __str__(self):
     def __iter__(self) -> typing.Iterator:
         """:return: A iterator that iterates over all parameter instances."""
         yield from self._params.values()
+
+    def completed(self) -> bool:
```

Comment:
`completed` is a little bit strange for me, maybe `filled` is a better name?

Reply:
I though about that, but chose `model.params.completed` because it's less ambiguous. `model.params.filled` doesn't necessarily convey the idea of **all** parameters being filled, but `model.params.completed` does. Same reason why you can't directly convert a numpy array into a bool, but have to use `numpy.ndarray.any` or `numpy.ndarray.all`.

Reply:
Not a big issue

Reply:
I chose names very carefully, so when you see a name change, there's definitely some consideration behind it.